python/copr/v3/proxies/project.py

```diff
@@ -154,7 +156,7 @@ def edit(self, ownername, projectname, chroots=None, description=None, instructi
              contact=None, additional_repos=None, unlisted_on_hp=None, enable_net=None,
              auto_prune=None, use_bootstrap_container=None, devel_mode=None,
              delete_after_days=None, multilib=None, module_hotfixes=None,
-             bootstrap=None, bootstrap_image=None, isolation=None,
+             bootstrap=None, bootstrap_image=None, isolation=None, follow_fedora_branching=True,
```

Comment:
I think this must be `follow_fedora_branching=True` otherwise running the edit action without specifying the `follow_fedora_branching` parameter, it would enable it for the project even if it was previously disabled.